easybuild/easyconfigs/g/GEM-library/GEM-library-20121106-022124_pre-release-2_Linux-x86_64.eb

```diff
@@ -0,0 +1,33 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Daoming Wang
+# Department of Genetics
+# University Medical Center Groningen
+
+easyblock = "Binary"
+
+name = 'GEM-library'
+version = '20121106-022124'
+versionsuffix = '_pre-release-2_Linux-x86_64'
+
+homepage = 'http://algorithms.cnag.cat/wiki/The_GEM_library'
```

Comment:
This homepage doesn't exist?

Maybe use https://sourceforge.net/projects/gemlibrary/ instead...